spec/support/jasmine-browser.json

```diff
@@ -2,7 +2,8 @@
   "srcDir": "public/assets",
   "srcFiles": [
     "**/*.js",
-    "!express/**/*.js"
+    "!express/**/*.js",
+    "!**/visualise-*.js"
```

Comment:
Do we have a note or card somewhere to remind us to add this back when possible?